src/core/annotation.js

```diff
@@ -987,6 +987,22 @@ class WidgetAnnotation extends Annotation {
     const vPadding = defaultPadding + Math.abs(descent) * fontSize;
     const defaultAppearance = this.data.defaultAppearance;
 
+    if (this.data.comb) {
```

Comment:
Since `comb` is a specific property of text widget annotations (and not other widget annotations), this code should live in the `TextWidgetAnnotation` class instead.